src/AWS/Lambda/Combinators.hs

```diff
@@ -32,6 +34,11 @@ dropEither = \case
      Right x -> return x
 
 
+-- | TODO
+liftIOEither :: MonadIO m => IO (Either String a) -> m a
```

Comment:
I feel like this function name is a bit too banal given that it can call `error`

Reply:
Agreed, I couldn't quite come up with something that was quite satisfying.  Maybe better to just inline it?  Not like there's a lot  going on!

Reply:
Inlining `either error pure <=< liftIO` is probably not too bad, but at actual use sites you might be able to rearrange it into something tidier.